machine-learning/lambda-v2/README.md

```diff
@@ -33,6 +39,8 @@ These sample components provide the following configuration parameters that can
 The following configuration is applicable for both the `com.lambda.DLRImageClassification` and 
 `com.lambda.DLRObjectDetection` inference components. 
 
+Since these inference components are lambda components, we can set the following configuration values only during the deployment. More info on setting configuration during deployment can be found [here](https://docs.aws.amazon.com/greengrass/v2/developerguide/update-component-configurations.html)
```

Comment:
I've rewritten these sentences in AWS style. How's the following?

> The `com.lambda.DLRImageClassification` and `com.lambda.DLRObjectDetection` inference components require the following configuration parameters. You specify these configuration values when you deploy the components. Because these component are Lambda function components, you can't set default configuration values in the component recipe. For more information, see [Update component configurations](https://docs.aws.amazon.com/greengrass/v2/developerguide/update-component-configurations.html).